src/iotjs_util.c

```diff
@@ -27,7 +27,10 @@
 
 iotjs_string_t iotjs_file_read(const char* path) {
   FILE* file = fopen(path, "rb");
-  IOTJS_ASSERT(file != NULL);
+  if (file == NULL) {
+    iotjs_string_t empty_content = iotjs_string_create();
+    return empty_content;
+  }
```

Comment:
Thank you for removing assert that could be false. I think we need to handle differently for actual empty content and `fopen` error. I found there is the only use case for iotjs_file_read for process.readsource. We need to add test case for this not-handled-till-now cases.